pkgs/by-name/op/opendungeons/package.nix

```diff
@@ -46,9 +46,14 @@ stdenv.mkDerivation {
   ];
 
   # source/utils/StackTraceUnix.cpp:122:2: error: #error Unsupported architecture.
```

Comment:
nit: maybe move this comment one line below, closer to the section that it belongs to.